### PR TITLE
tests: Fix tests failing with 1.3 devsim

### PR DESCRIPTION
**Pull request by sfricke-samsung**
Found when running MockICD with a 1.3 device via devsim layer

### CODE REVIEW

File: tests/vklayertests_command.cpp [hunk 1]
```diff
@@ -5865,6 +5865,14 @@ TEST_F(VkLayerTest, MultiDrawFeatures) {
     ASSERT_NO_FATAL_FAILURE(InitState());
     ASSERT_NO_FATAL_FAILURE(InitRenderTarget());
 
+    auto multi_draw_props = LvlInitStruct<VkPhysicalDeviceMultiDrawPropertiesEXT>();
+    auto pd_props2 = LvlInitStruct<VkPhysicalDeviceProperties2>(&multi_draw_props);
+    vk::GetPhysicalDeviceProperties2(gpu(), &pd_props2);
+    if (multi_draw_props.maxMultiDrawCount == 0) {
+        // If using MockICD and devsim the value might be zero'ed casue false errors
```
Comment (Tony-LunarG) [inline tests/vklayertests_command.cpp:5872]: Typo casue->cause

**Comment by Tony-LunarG** [comment]
@sfricke-samsung out of curiosity, what 1.3 devsim json did you use?

**Comment by sfricke-samsung** [comment]
@Tony-LunarG I was using
https://vulkan.gpuinfo.org/displayreport.php?id=13975#device

I am not sure how CI passes to be honest, because all these failures were mainly from have a limit of `0` because devsim can't get the extended properties values out

**Comment by Tony-LunarG** [comment]
I'm seeing more failures than these, but these changes fix the tests they intend to.

**Review by Tony-LunarG — APPROVED** [review]
LGTM

**Pull request merged**